pool/hub.go

```diff
@@ -24,6 +24,7 @@ import (
 	"decred.org/dcrwallet/rpc/walletrpc"
 	"github.com/decred/dcrd/chaincfg/chainhash"
 	"github.com/decred/dcrd/chaincfg/v3"
+	"github.com/decred/dcrd/dcrjson/v4"
```

Comment:
```suggestion
	"github.com/decred/dcrd/dcrjson/v3"
```

We are using `rpcclient/v6` which returns responses from `dcrjson/v3`, not `v4`